schema/addresses/address.yaml

```diff
@@ -71,6 +71,14 @@ properties:     # JSON Schema: Top-level object properties.
         items: { "$ref": "#/$defs/propertyContainers/addressLevelContainer" }
         minItems: 1
         maxItems: 5
+      postal_city:
```

Comment:
`postal_locality`?

Reply:
@vcschapp I asked around I think `postal_city` is the common phrase used for addresses for this.